packages/itinerary-body/src/TransitLegBody/index.tsx

```diff
@@ -34,13 +35,16 @@ interface Props {
   AlertBodyIcon?: FunctionComponent;
   AlertToggleIcon?: FunctionComponent;
   alwaysCollapseAlerts: boolean;
+  arrivalText: string;
+  arrivalTripId: string;
```

Comment:
In an itinerary with multiple transit legs, did you intend to render a different arrival time for each transit leg? Your latest changes don't support that as written because the container for `TransitLegBody`, which is `PlaceRow`, never sets the arrival props.